.github/workflows/k8s-setup-otc.yml

```diff
@@ -80,3 +84,27 @@ jobs:
       - name: Apply Storage Classes
         run: |
           kubectl apply -f ${{ inputs.storage_class_yaml }}
+
+  deploy-kyverno:
+    runs-on: [self-hosted, docker]
+    if: contains('["acpoppe","KlausNie,"kupeliorhun""]', github.triggering_actor)
```

Comment:
are you sure this works? with the double `"`?

Reply:
hmmm, nice catch